MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -4578,7 +4579,8 @@ public string GetQsvHwVidDecoder(EncodingJobInfo state, EncodingOptions options,
             var hwSurface = (isIntelDx11OclSupported || isIntelVaapiOclSupported)
                 && _mediaEncoder.SupportsFilter("alphasrc");
 
-            var is8bitSwFormatsQsv = string.Equals("yuv420p", videoStream.PixelFormat, StringComparison.OrdinalIgnoreCase);
+            var is8bitSwFormatsQsv = string.Equals("yuv420p", videoStream.PixelFormat, StringComparison.OrdinalIgnoreCase)
+                || string.Equals("yuvj420p", videoStream.PixelFormat, StringComparison.OrdinalIgnoreCase);
```

Comment:
Please fix indentation for the next 4 additions too
```suggestion
                                     || string.Equals("yuvj420p", videoStream.PixelFormat, StringComparison.OrdinalIgnoreCase);
```

Reply:
Done.